From 9850414c0425aebf038faf418f331df2d6ad573b Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 5 Nov 2021 08:53:26 +0000
Subject: [PATCH] GCC: make gfx::Point::operator== constexpr

blink::IntRect::operator== is constexpr. Therefore,
gfx::Point::operator== needs to be constexpr as well.
---
 ui/gfx/geometry/point.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gfx/geometry/point.h b/ui/gfx/geometry/point.h
index 71c5a5b..55b3220 100644
--- a/ui/gfx/geometry/point.h
+++ b/ui/gfx/geometry/point.h
@@ -95,7 +95,7 @@ class GEOMETRY_EXPORT Point {
   int y_;
 };
 
-inline bool operator==(const Point& lhs, const Point& rhs) {
+constexpr bool operator==(const Point& lhs, const Point& rhs) {
   return lhs.x() == rhs.x() && lhs.y() == rhs.y();
 }
 
-- 
2.32.0

